### DISCUSSION TITLE
Buildbase deployed app as PWA (Progressive Web App)

### DISCUSSION THREAD

**Discussion by Sunraj751** [Budibase/budibase#14000] [Help] [answered]
Hi I am trying to build a frontend for an MVP, and was wondering if it's possible to deploy a web app made using buildbase as a Progressive Web App ? (_aka allowing people to download the web app on home screen)_ **OR** if at all possible deploy it as a native mobile app ?

**Comment by ConorWebb96** [comment, marked as answer]
Hey @Sunraj751,

A user within our community discord wrote up a summary of how they managed to turn a Budibase app into a PWA. You can find the information for this [here](https://discord.com/channels/733030666647765003/1014909112565379163/1213537593526980699).
